cpg_workflows/__init__.py

```diff
@@ -7,6 +7,217 @@
+
+# VQSR (relative to broad_ref)
+dbsnp_vcf = 'Homo_sapiens_assembly38.dbsnp138.vcf'
+dbsnp_vcf_index = 'Homo_sapiens_assembly38.dbsnp138.vcf.idx'
+hapmap_vcf = 'hapmap_3.3.hg38.vcf.gz'
+hapmap_vcf_index = 'hapmap_3.3.hg38.vcf.gz.tbi'
+omni_vcf = '1000G_omni2.5.hg38.vcf.gz'
+omni_vcf_index = '1000G_omni2.5.hg38.vcf.gz.tbi'
+one_thousand_genomes_vcf = '1000G_phase1.snps.high_confidence.hg38.vcf.gz'
+one_thousand_genomes_vcf_index = '1000G_phase1.snps.high_confidence.hg38.vcf.gz.tbi'
+mills_vcf = 'Mills_and_1000G_gold_standard.indels.hg38.vcf.gz'
+mills_vcf_index = 'Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi'
+axiom_poly_vcf = 'Axiom_Exome_Plus.genotypes.all_populations.poly.hg38.vcf.gz'
+axiom_poly_vcf_index = 'Axiom_Exome_Plus.genotypes.all_populations.poly.hg38.vcf.gz.tbi'
+
+# Genome contamination check
+genome_contam_ud =  'contamination-resources/1000g/1000g.phase3.100k.b38.vcf.gz.dat.UD'
+genome_contam_bed = 'contamination-resources/1000g/1000g.phase3.100k.b38.vcf.gz.dat.bed'
+genome_contam_mu =  'contamination-resources/1000g/1000g.phase3.100k.b38.vcf.gz.dat.mu'
+# Exome contamination check
+exome_contam_ud = 'contamination-resources/1000g/whole_exome_illumina_coding_v1.Homo_sapiens_assembly38.1000g.contam.UD'
+exome_contam_bed = 'contamination-resources/1000g/whole_exome_illumina_coding_v1.Homo_sapiens_assembly38.1000g.contam.bed'
+exome_contam_mu = 'contamination-resources/1000g/whole_exome_illumina_coding_v1.Homo_sapiens_assembly38.1000g.contam.mu'
+
+## GnomAD resources for Hail Query
+[references.gnomad]
+prefix = 'gnomad/v0'
+# All paths below relative to prefix.
+tel_and_cent_ht = 'telomeres_and_centromeres/hg38.telomeresAndMergedCentromeres.ht'
+lcr_intervals_ht = 'lcr_intervals/LCRFromHengHg38.ht'
+seg_dup_intervals_ht = 'seg_dup_intervals/GRCh38_segdups.ht'
+clinvar_ht = 'clinvar/clinvar_20190923.ht'
+hapmap_ht = 'hapmap/hapmap_3.3.hg38.ht'
+kgp_omni_ht = 'kgp/1000G_omni2.5.hg38.ht'
+kgp_hc_ht = 'kgp/1000G_phase1.snps.high_confidence.hg38.ht'
+mills_ht = 'mills/Mills_and_1000G_gold_standard.indels.hg38.ht'
+
+[references.seqr]
+prefix = 'seqr/v0-1'
+combined_reference = 'combined_reference_data_grch38-2.0.4.ht'
+clinvar = 'clinvar.GRCh38.ht'
+
+[references.validation]
+prefix = 'validation'
+[references.validation.syndip]
+truth = 'syndip/truth/full.38.20180222.vcf.gz'
+regions = 'syndip/regions/syndip.b38_20180222.bed'
+[references.validation.HG001]
+truth = 'giab/truth/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.vcf.gz'
+regions = 'giab/regions/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_nosomaticdel_noCENorHET7.bed'
+
+# Map internally used validation sample external_id to truth sample names
+[validation.sample_map]
+HG001_NA12878 = 'HG001'
+SYNDIP = 'syndip'
+
+[hail]
+delete_scratch_on_exit = false
+"""
+
+
+f = tempfile.NamedTemporaryFile(delete=False, mode='w', suffix='.toml')
```

Comment:
At this stage could we read the file specified at cpg_workflows/defaults.toml and then write it to this temporary file as a work-around without needing to hardcode the .toml in this init file?

Reply:
That's the problem I'm trying to workaround with this PR - we can't read it because it wouldn't be uploaded to dataproc along with python files. 

This would probably solve this though: https://github.com/populationgenomics/hail/pull/249